src/wav2vec2/config.py

```diff
@@ -30,12 +31,22 @@ class Wav2Vec2Config:
     strides: list = field(default_factory=lambda: [5, 2, 2, 2, 2, 2, 2])
     conv_bias: bool = False
 
+    # spec augmentation arguments
+    apply_spec_augment: bool = False  # TODO: fix this
+    mask_time_prob: float = 0.05
+    mask_time_length: int = 10
+
     def __post_init__(self):
         if not (len(self.filter_sizes) == len(self.kernal_sizes) == len(self.strides)):
-            raise ValueError("len of filter_sizes, kernal_sizes, strides must much")
+            raise ValueError("len of filter_sizes, kernal_sizes, strides must match")
```

Comment:
Use similar casing throughout. If possible try concluding the sentence with ".".